<!DOCTYPE html>
<html lang="en">

<head>
    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-79519485-2"></script>
    <script>
    window.dataLayer = window.dataLayer || [];

    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());

    gtag('config', 'UA-79519485-2');
    </script>
    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="Fengchao Yu's Homepage">
    <meta name="author" content="Fengchao">
    <title>Fengchao Yu</title>
    <link href="css/bootstrap.min.css" rel="stylesheet" type="text/css">
    <link href="css/fontawesome-all.min.css" rel="stylesheet" type="text/css">
    <link href="css/academicons.min.css" rel="stylesheet" type="text/css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/devicon/2.2/devicon.min.css" />
    <link href="css/resume.min.css" rel="stylesheet" type="text/css">
    <link href="css/saira_extra_condensed.css" rel="stylesheet" type="text/css">
    <link href="css/open_sans.css" rel="stylesheet" type="text/css">
</head>

<body id="page-top">
    <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
        <a class="navbar-brand js-scroll-trigger" href="#page-top">
            <span class="d-block d-lg-none">Fengchao Yu</span>
            <span class="d-none d-lg-block">
                <img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="img/profile.jpg" alt="">
            </span>
        </a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
            <ul class="navbar-nav">
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#about">About</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#experience">Research Experience</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#publications">Journal Publications</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#conference">Presentations</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#awards">Awards</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#activities">Professional Activities</a>
                </li>
            </ul>
        </div>
    </nav>
    <div class="container-fluid p-0">
        <section class="resume-section p-3 p-lg-5 d-flex d-column" id="about">
            <div class="my-auto">
                <h1 class="mb-0">Fengchao
                    <span class="text-primary">Yu</span>
                </h1>
                <div class="subheading mb-5">University of Michigan, Ann Arbor, Michigan, USA</div>
                <p class="mb-5">I am a research investigator from <a href="https://www.nesvilab.org/" target="_blank">Alexey Nesvizhskii's lab</a> at <a href="https://umich.edu/" target="_blank">University of Michigan</a>. My research interests include Proteomics, Bioinformatics, computational biology, machine learning, and statistics. I am the leading developer and maintainer of <a href="https://fragpipe.nesvilab.org/" target="_blank">FragPipe</a>, <a href="https://msfragger.nesvilab.org/" target="_blank">MSFragger</a>, and <a href="https://ionquant.nesvilab.org/" target="_blank">IonQuant</a>.</p>
                <ul class="list-inline list-social-icons mb-0">
                    <li class="list-inline-item">
                        <a href="https://orcid.org/0000-0002-7695-3698" target="_blank">
                            <span class="fa-stack fa-lg">
                                <i class="ai ai-orcid ai-2x"></i>
                            </span>
                        </a>
                    </li>
                    <li class="list-inline-item">
                        <a href="https://www.linkedin.com/in/fengchao/" target="_blank">
                            <span class="fa-stack fa-lg">
                                <i class="fab fa-linkedin fa-2x"></i>
                            </span>
                        </a>
                    </li>
                    <li class="list-inline-item">
                        <a href="https://github.com/fcyu" target="_blank">
                            <span class="fa-stack fa-lg">
                                <i class="fab fa-github fa-2x"></i>
                            </span>
                        </a>
                    </li>
                    <li class="list-inline-item">
                        <a href="https://scholar.google.com/citations?user=eqDf4X4AAAAJ&hl=en" target="_blank">
                            <span class="fa-stack fa-lg">
                                <i class="ai ai-google-scholar ai-2x"></i>
                            </span>
                        </a>
                    </li>
                    <li class="list-inline-item">
                        <a href="mailto:yufe@umich.edu" data-toogle="tooltip" title="yufe@umich.edu">
                            <span class="fa-stack fa-lg">
                                <i class="far fa-envelope fa-2x"></i>
                            </span>
                        </a>
                    </li>
                </ul>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="experience">
            <div class="my-auto">
                <h2 class="mb-5">Research Experience</h2>
                <div class="resume-item d-flex flex-column flex-md-row mb-5">
                    <div class="resume-content mr-auto">
                        <h3 class="mb-0">Department of Pathology, University of Michigan</h3>
                        <div class="subheading mb-3">Research Investigator</div>
                    </div>
                    <div class="resume-date text-md-right">
                        <span class="text-primary">May 2021 - present</span>
                    </div>
                </div>
                <div class="resume-item d-flex flex-column flex-md-row mb-5">
                    <div class="resume-content mr-auto">
                        <h3 class="mb-0">Department of Pathology, University of Michigan</h3>
                        <div class="subheading mb-3">Research Fellow</div>
                    </div>
                    <div class="resume-date text-md-right">
                        <span class="text-primary">January 2019 - April 2021</span>
                    </div>
                </div>
                <div class="resume-item d-flex flex-column flex-md-row mb-5">
                    <div class="resume-content mr-auto">
                        <h3 class="mb-0">Department of Electronic and Computer Engineering, The Hong Kong University of Science and Technology</h3>
                        <div class="subheading mb-3">Research Associate</div>
                    </div>
                    <div class="resume-date text-md-right">
                        <span class="text-primary">June 2017 - December 2018</span>
                    </div>
                </div>
                <div class="resume-item d-flex flex-column flex-md-row">
                    <div class="resume-content mr-auto">
                        <h3 class="mb-0">Division of Biomedical Engineering, The Hong Kong University of Science and Technology</h3>
                        <div class="subheading mb-3">Doctor of Philosophy</div>
                    </div>
                    <div class="resume-date text-md-right">
                        <span class="text-primary">September 2012 - June 2017</span>
                    </div>
                </div>
                <div class="resume-item d-flex flex-column flex-md-row">
                    <div class="resume-content mr-auto">
                        <h3 class="mb-0">Department of Optical Engineering, Zhejiang University</h3>
                        <div class="subheading mb-3">Master of Engineering</div>
                    </div>
                    <div class="resume-date text-md-right">
                        <span class="text-primary">September 2009 - March 2012</span>
                    </div>
                </div>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="publications">
            <div class="my-auto">
                <h2 class="mb-5">Journal Publications</h2>
                <ol class="mb-0">
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Park, J., <string>Yu, F.</string>, Fulcher, J.M., Williams, S.M., Engbrecht, K., Moore, R.J., Clair, G.C., Petyuk, V., Nesvizhskii, A.I., &amp; Zhu, Y., (2023). Evaluating Linear Ion Trap for MS3-Based Multiplexed Single-Cell Proteomics. <em>Analytical Chemistry</em>, in press.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.analchem.2c03739" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.analchem.2c03739" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Chang, H.-Y., Haynes, S. E., <strong>Yu, F.</strong>, &amp; Nesvizhskii, A. I., (2022). Implementing the MSFragger Search Engine as a Node in Proteome Discoverer. <em>Journal of Proteome Research</em>, in press.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.2c00485" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.2c00485" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Kacen, A., Javitt, A., Kramer, M., Morgenstern, D., Tsaban, T., Shmueli, M., Teo, G. C., Leprevost, F. V., Barnea, E., <strong>Yu, F.</strong>, Admon, A., Eisenbach, L., Samuels, Y., Schueler-Furman, O., Levin, Y., &amp; Nesvizhskii, A. (2022). Post-translational modifications reshape the antigenic landscape of MHC I-immunopeptidome in tumors. <em>Nature Biotechnology</em>, in press.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41587-022-01464-2" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41587-022-01464-2" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Demichev, V., Szyrwiel, L., <strong>Yu, F.</strong>, Teo, G. C., Rosenberger, G., Niewienda, A., Ludwig, D., Decker, J., Kaspar-Schoenefeld, S., Lilley, K., Mulleder, M., Nesvizhskii, A., &amp; Ralser, M. (2022). dia-PASEF data analysis using FragPipe and DIA-NN for deep proteomics of low sample amounts. <em>Nature Communications</em>, 13, 3944.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41467-022-31492-0" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-022-31492-0" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Desai, H., Yan, T., <strong>Yu, F.</strong>, Sun, A., Villanueva, M., Nesvizhskii, A., &amp; Backus, K. (2022). SP3-enabled Rapid and High Coverage Chemoproteomic Identification of Cell-State Dependent Redox-Sensitive Cysteines. <em>Molecular &amp; Cellular Proteomics</em>, 100218.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.mcpro.2022.100218" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.mcpro.2022.100218" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Polasky, D., Geiszler, D., <strong>Yu, F.</strong> &amp; Nesvizhskii, A. (2022). Multi-attribute Glycan Identification and FDR Control for Glycoproteomics. <em>Molecular &amp; Cellular Proteomics</em>, 100205.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.mcpro.2022.100205" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.mcpro.2022.100205" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Moradi, A., Dai, S., Wong, E.O.Y., Zhu, G., <strong>Yu, F.</strong>, Lam, H.M., Wang, Z., Burlingame, A., Lin, C., Afsharifar, A. &amp; Yu, W. (2021). Isotopically Dimethyl Labeling-Based Quantitative Proteomic Analysis of Phosphoproteomes of Soybean Cultivars. <em>Biomolecules</em>, 11(8), p.1218.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.3390/biom11081218" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.3390/biom11081218" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Petyuk, V.A., Yu, L., Olson, H.M., <strong>Yu, F.</strong>, Clair, G., Qian, W.J., Shulman, J.M. &amp; Bennett, D.A. (2021). Proteomic Profiling of the Substantia Nigra to Identify Determinants of Lewy Body Pathology and Dopaminergic Neuronal Loss. <em>Journal of Proteome Research</em>, 20(5), 2266-2282.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.0c00747" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.0c00747" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Haynes, S., &amp; Nesvizhskii, A. (2021). IonQuant enables accurate and sensitive label-free quantification with FDR-controlled match-between-runs. <em>Molecular &amp; Cellular Proteomics</em>, 20, 100077.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.mcpro.2021.100077" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.mcpro.2021.100077" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Huang, C., Chen, L., Savage, S. R., Eguez, R. V., Dou, Y., Li, Y., ... & Piehowski, P. D. (2021). Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma. <em>Cancer Cell</em>, 39(3), 361-379.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.ccell.2020.12.007" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ccell.2020.12.007" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Geiszler, D., Kong, A., Avtonomov, D., <strong>Yu, F.</strong>, Leprevost, F., &amp; Nesvizhskii, A. (2021). PTM-Shepherd: analysis and summarization of post-translational and chemical modifications from open search results. <em>Molecular &amp; Cellular Proteomics</em>, 20, 100018.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1074/mcp.TIR120.002216" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1074/mcp.TIR120.002216" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Dai, J.*, <strong>Yu, F.*</strong>, Chen Zhou, &amp; Yu, W. (2021). Understanding the limit of open search in the identification of peptides with post-translational modifications — A simulation-based study. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em>, 18(6), 2884-2890. (* co-first author)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1109/TCBB.2020.2991207" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1109/TCBB.2020.2991207" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Teo, G. C., Polasky, D., <strong>Yu, F.</strong>, &amp; Nesvizhskii, A. (2020). Fast deisotoping algorithm and its implementation in the MSFragger search engine. <em>Journal of Proteome Research</em>, 20(1), 498-505.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.0c00544" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.0c00544" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Polasky, D., <strong>Yu, F.</strong>, Teo, G. C., &amp; Nesvizhskii, A. (2020). Fast and comprehensive N- and O-glycoproteomics analysis with MSFragger-Glyco. <em>Nature Methods</em>, 17(11), 1125–1132.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41592-020-0967-9" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41592-020-0967-9" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Teo, G. C., Kong, A., Avtonomov, D., Geiszler, D., &amp; Nesvizhskii, A. (2020). Identification of modified peptides using localization-aware open search. <em>Nature Communications</em>, 11, 4065.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41467-020-17921-y" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-020-17921-y" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.*</strong>, Haynes, S.*, Teo, G. C., Avtonomov, D., Polasky, D., &amp; Nesvizhskii, A. (2020). Fast quantitative analysis of timsTOF PASEF data with MSFragger and IonQuant. <em>Molecular &amp; Cellular Proteomics</em>, 19(9), 1575-1585. (* co-first author)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1074/mcp.TIR120.002048" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1074/mcp.TIR120.002048" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Liu, S.*, <strong>Yu, F.*</strong>, Hu, Q., Wang, T., Yu, L., Du, S., Yu, W., &amp; Li, N. (2018). Development of in planta chemical cross-linking-based quantitative interactomics in Arabidopsis. <em>Journal of Proteome Research</em>, 17(9), 3195-3213. (* co-first author)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.8b00320" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.8b00320" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Dai, J.*, Jiang, W.*, <strong>Yu, F.*</strong>, &amp; Yu, W. (2018). Xolik: finding cross-linked peptides with maximum paired scores in linear time. <em>Bioinformatics</em>, 35(2), 251-257. (* co-first author)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1093/bioinformatics/bty526" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/bty526" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Liu, S.*, <strong>Yu, F.*</strong>, Yang, Z., Wang, T., Xiong, H., Chang, C., Yu, W., &amp; Li, N. (2018). Establishment of dimethyl labeling-based quantitative acetylproteomics in Arabidopsis. <em>Molecular &amp; Cellular Proteomics</em>, 17(5), 1010-1027. (* co-first author)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1074/mcp.RA117.000530" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1074/mcp.RA117.000530" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (2017). Exhaustively identifying cross-linked peptides with a linear computational complexity. <em>Journal of Proteome Research</em>, 16(10), 3942-3952.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.7b00338" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.7b00338" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (2016). PIPI: PTM-invariant peptide identification using coding method. <em>Journal of Proteome Research</em>, 15(12), 4423-4435.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.6b00485" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.6b00485" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (2016). ECL: an exhaustive search tool for the identification of cross-linked peptides using whole database. <em>BMC Boinformatics</em>, 17(1), 217.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1186/s12859-016-1073-y" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1186/s12859-016-1073-y" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Zhu, X.*, <strong>Yu, F.*</strong>, Yang, Z., Liu, S., Dai, C., Lu, X., Liu, C., Yu, W., &amp; Li, N. (2016). In planta chemical cross‐linking and mass spectrometry analysis of protein structure and interaction in Arabidopsis. <em>Proteomics</em>,
                                16(13), 1915-1927. (* co-first author)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1002/pmic.201500310" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1002/pmic.201500310" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Liu, H., Hu, Z., &amp; Shi, P. (2012). Graphics processing unit (GPU)-accelerated particle filter framework for positron emission tomography image reconstruction. <em>JOSA A</em>, 29(4), 637-643.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1364/JOSAA.29.000637" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1364/JOSAA.29.000637" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                </ol>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="conference">
            <div class="my-auto">
                <h2 class="mb-5">Presentations</h2>
                <ol class="mb-0">
                    <li>
                        <strong>Yu, F.</strong>, Teo, G.C., da Veiga Leprevost, F., Polasky, D., Geiszler, D., Yang, K., Kai, Li., &amp; Nesvizhskii, A. (2022, October). FragPipe enables one-stop data analysis for bottom-up proteomics. <em>2022 CASMS virtual conference (CASMS 2022)</em>. (poster presentation, best poster award).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Teo, G.C., Avtonomov. D., Haynes, S., da Veiga Leprevost, F., Geiszler, D., Polasky, D., Yang, K., &amp; Nesvizhskii, A. (2022, July). FragPipe in headless mode enables seamlessly analyzing LC-MS data in Galaxy platform. <em>2022 Galaxy Community Conference (GCC 2022)</em>. (oral presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Kong, A., Patil, S., Avtonomov, D., &amp; Nesvizhskii, A. (2022, June). Pepcentric enables fast peptide searching against public data repositories. <em>70th ASMS Conference on Mass Spectrometry and Allied Topics (ASMS 2022)</em>. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Yang, K., Demichev, V., Haynes, S., Ralser, M., &amp; Nesvizhskii, A. (2022, March). One-stop DIA data analysis using MSFragger-DIA coupled with FragPipe. <em>18th Annual US HUPO Conference (US HUPO 2022)</em>. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Teo, G. C., Haynes, S., Li, G. X., &amp; Nesvizhskii, A. (2021, November). Direct peptide identification from DIA data with MSFragger DIA. <em>69th ASMS Conference on Mass Spectrometry and Allied Topics (ASMS 2021)</em>. (oral presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Haynes, S., &amp; Nesvizhskii, A. (2021, March). False discovery rate-controlled match-between-runs enables accurate and sensitive label free quantification. <em>17th Annual US HUPO Conference (US HUPO 2021)</em>. (poster presentation), virtual conference.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Teo, G. C., Kong, A., Haynes, S., Avtonomov, D., Geiszler, D., &amp; Nesvizhskii, A. (2020, June). Improved identification of modified peptides using localization-aware open search. <em>28th Conference on Intelligent Systems for Molecular Biology (ISMB 2020)</em>. (poster presentation), virtual conference.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Haynes, S., Avtonomov, D., Kong, A., Leprevost, F., &amp; Nesvizhskii, A. (2020, June). Breaking the logjam: fast peptide identification and quantification in timsTOF PASEF data. <em>68th ASMS Conference on Mass Spectrometry and Allied Topics (ASMS 2020)</em>. (poster presentation), virtual conference.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Haynes, S., Teo, G. C., Kong, A., Avtonomov, D., Leprevost, F., Chang, H.-Y., Geiszler, D., Polasky, D., &amp; Nesvizhskii, A. (2020, March). Fast and quantitative analysis of timsTOF PASEF data with MSFragger and IonQuant. <em>16th Annual US HUPO Conference (US HUPO 2020)</em>. (poster presentation), virtual conference.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Teo, G. C., Kong, A., Leprevost, F., Avtonomov, D., Chang, H., Geiszler, D., Haynes, S., Polasky, D., &amp; Nesvizhskii, A. (2019, October). MSFragger: fast and sensitive peptide identification in diverse proteomic datasets. <em>18th Human Proteome Organization World Congress (HUPO 2019)</em>. (poster presentation), Adelaide, Australia.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Teo, G. C., Kong, A., Leprevost, F., Chang, H., &amp; Nesvizhskii, A. (2019, June). Comparison of open search tools. <em>67th ASMS Conference on Mass Spectrometry and Allied Topics (ASMS 2019)</em>. (poster presentation), Atlanta, Georgia.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (2018, August). PTM-invariant peptide identification. <em>5th China Workshop on Computational Proteomics (CNCP 2018)</em>. (oral presentation), Beijing, China.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (2017, August). PIPI: PTM-invariant peptide identification using coding method. <em>Post-translational Modification Networks, Gordon Research Conference</em>. (poster presentation), Hong Kong, China.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (2017, August). PIPI: PTM-invariant peptide identification using coding method. <em>11th International Conference on Computational Systems Biology (ISB 2017)</em>. (oral presentation), Shenzhen, China.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (2017, May). Exhaustively identifying cross-linked peptides with a linear complexity. <em>21st Annual International Conference on Research in Computational Molecular Biology (RECOMB 2017)</em>. (poster presentation), Hong Kong, China.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Liu, H., &amp; Shi, P. (2012, January). PET image reconstruction based on particle filter framework. <em>In Biomedical and Health Informatics (BHI), 2012 IEEE-EMBS International Conference on</em> (pp. 851-853). IEEE.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Liu, H., &amp; Shi, P. (2011, September). PET image reconstruction: GPU-accelerated particle filter framework. <em>In Image Processing (ICIP), 2011 18th IEEE International Conference on</em> (pp. 417-420). IEEE.
                    </li>
                </ol>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="awards">
            <div class="my-auto">
                <h2 class="mb-5">Awards</h2>
                <ul class="fa-ul mb-0">
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>CASMS (Chinese American Society for Mass Spectrometry) Best Poster Award (2022)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>US HUPO (United States Human Proteome Organization) Travel Stipend Award (2020)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>UGC Research Travel Grant Award (2018-19)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>UGC Research Travel Grant Award (2018-19)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>UGC Research Travel Grant Award (2017-18)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>UGC Research Travel Grant Award (2016-17)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>Graduate Student Research Award Program of HKMHDIA (Hong Kong Medical &amp; Healthcare Device Industries Association Limited) (2015-16)
                    </li>
                </ul>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="activities">
            <div class="my-auto">
                <h2 class="mb-5">Professional Activities</h2>
                <h3 class="mb-5">Journal reviewer</h2>
                    <style type="text/css">
                    .tg {
                        border-collapse: collapse;
                        border-spacing: 0;
                    }

                    .tg td {
                        border-color: black;
                        border-style: solid;
                        border-width: 1px;
                        font-family: Arial, sans-serif;
                        font-size: 14px;
                        overflow: hidden;
                        padding: 10px 5px;
                        word-break: normal;
                    }

                    .tg th {
                        border-color: black;
                        border-style: solid;
                        border-width: 1px;
                        font-family: Arial, sans-serif;
                        font-size: 14px;
                        font-weight: normal;
                        overflow: hidden;
                        padding: 10px 5px;
                        word-break: normal;
                    }

                    .tg .tg-7zrl {
                        text-align: left;
                        vertical-align: bottom
                    }
                    </style>
                    <table class="tg">
                        <thead>
                            <tr>
                                <th class="tg-7zrl">Journal</th>
                                <th class="tg-7zrl">Reviewed</th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td class="tg-7zrl">Nature Communications</td>
                                <td class="tg-7zrl">2</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">Nature Computational Science</td>
                                <td class="tg-7zrl">3</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">Briefings in Bioinformatics</td>
                                <td class="tg-7zrl">12</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">Bioinformatics</td>
                                <td class="tg-7zrl">11</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">Journal of Proteome Research</td>
                                <td class="tg-7zrl">12</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">Proteomics</td>
                                <td class="tg-7zrl">5</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">Journal of Proteomics</td>
                                <td class="tg-7zrl">1</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">Analytical Chemistry</td>
                                <td class="tg-7zrl">5</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">ACS Omega</td>
                                <td class="tg-7zrl">1</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">BMC Bioinformatics</td>
                                <td class="tg-7zrl">8</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">BMC Genomics</td>
                                <td class="tg-7zrl">1</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">Rapid Communications in Mass Spectrometry</td>
                                <td class="tg-7zrl">1</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">Bioinformatics Advances</td>
                                <td class="tg-7zrl">3</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">International Journal of Mass Spectrometry</td>
                                <td class="tg-7zrl">2</td>
                            </tr>
                            <tr>
                                <td class="tg-7zrl">total:</td>
                                <td class="tg-7zrl">67</td>
                            </tr>
                        </tbody>
                    </table>
            </div>
        </section>
    </div>
    <script src="js/jquery-3.3.1.min.js"></script>
    <script src="js/jquery.easing.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <script src="js/popper.min.js"></script>
    <script src="js/resume.min.js"></script>
</body>

</html>
