<!DOCTYPE html>
<html lang="en">

<head>
    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-79519485-2"></script>
    <script>
    window.dataLayer = window.dataLayer || [];

    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());

    gtag('config', 'UA-79519485-2');
    </script>
    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="Fengchao Yu's Homepage">
    <meta name="author" content="Fengchao">
    <title>Fengchao Yu</title>
    <link href="css/bootstrap.min.css" rel="stylesheet" type="text/css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
    <link href="css/academicons.min.css" rel="stylesheet" type="text/css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/devicon/2.2/devicon.min.css" />
    <link href="css/resume.min.css" rel="stylesheet" type="text/css">
    <link href="css/saira_extra_condensed.css" rel="stylesheet" type="text/css">
    <link href="css/open_sans.css" rel="stylesheet" type="text/css">
</head>

<body id="page-top">
    <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
        <a class="navbar-brand js-scroll-trigger" href="#page-top">
            <span class="d-block d-lg-none">Fengchao Yu</span>
            <span class="d-none d-lg-block">
                <img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="img/profile.jpg" alt="">
            </span>
        </a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
            <ul class="navbar-nav">
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#about">About</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#experience">Research Experience</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#publications">Journal Publications</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#conference">Presentations</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#awards">Awards</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#activities">Professional Activities</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#teaching">Teaching Activities</a>
                </li>
            </ul>
        </div>
    </nav>
    <div class="container-fluid p-0">
        <section class="resume-section p-3 p-lg-5 d-flex d-column" id="about">
            <div class="my-auto">
                <h1 class="mb-0">Fengchao
                    <span class="text-primary">Yu</span>
                </h1>
                <div class="subheading mb-5">University of Michigan, Ann Arbor, Michigan, USA</div>
                <p class="mb-5">I am a research investigator from <a href="https://www.nesvilab.org/" target="_blank">Alexey Nesvizhskii's lab</a> at <a href="https://umich.edu/" target="_blank">University of Michigan</a>. My research interests include Proteomics, Bioinformatics, computational biology, machine learning, and statistics. I am the leading developer and maintainer of <a href="https://fragpipe.nesvilab.org/" target="_blank">FragPipe</a>, <a href="https://msfragger.nesvilab.org/" target="_blank">MSFragger</a>, and <a href="https://ionquant.nesvilab.org/" target="_blank">IonQuant</a>.</p>
                <ul class="list-inline list-social-icons mb-0">
                    <li class="list-inline-item">
                        <a href="https://orcid.org/0000-0002-7695-3698" target="_blank">
                            <span class="fa-stack fa-lg">
                                <i class="ai ai-orcid ai-2x"></i>
                            </span>
                        </a>
                    </li>
                    <li class="list-inline-item">
                        <a href="https://www.linkedin.com/in/fengchao/" target="_blank">
                            <span class="fa-stack fa-lg">
                                <i class="fab fa-linkedin fa-2x"></i>
                            </span>
                        </a>
                    </li>
                    <li class="list-inline-item">
                        <a href="https://github.com/fcyu" target="_blank">
                            <span class="fa-stack fa-lg">
                                <i class="fab fa-github fa-2x"></i>
                            </span>
                        </a>
                    </li>
                    <li class="list-inline-item">
                        <a href="https://scholar.google.com/citations?user=eqDf4X4AAAAJ&hl=en" target="_blank">
                            <span class="fa-stack fa-lg">
                                <i class="ai ai-google-scholar ai-2x"></i>
                            </span>
                        </a>
                    </li>
                    <li class="list-inline-item">
                        <a href="mailto:yufe@umich.edu" data-toogle="tooltip" title="yufe@umich.edu">
                            <span class="fa-stack fa-lg">
                                <i class="far fa-envelope fa-2x"></i>
                            </span>
                        </a>
                    </li>
                </ul>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="experience">
            <div class="my-auto">
                <h2 class="mb-5">Research Experience</h2>
                <div class="resume-item d-flex flex-column flex-md-row mb-5">
                    <div class="resume-content mr-auto">
                        <h3 class="mb-0">Department of Pathology, University of Michigan</h3>
                        <div class="subheading mb-3">Research Investigator</div>
                    </div>
                    <div class="resume-date text-md-right">
                        <span class="text-primary">May 2021 - present</span>
                    </div>
                </div>
                <div class="resume-item d-flex flex-column flex-md-row mb-5">
                    <div class="resume-content mr-auto">
                        <h3 class="mb-0">Department of Pathology, University of Michigan</h3>
                        <div class="subheading mb-3">Research Fellow</div>
                    </div>
                    <div class="resume-date text-md-right">
                        <span class="text-primary">January 2019 - April 2021</span>
                    </div>
                </div>
                <div class="resume-item d-flex flex-column flex-md-row mb-5">
                    <div class="resume-content mr-auto">
                        <h3 class="mb-0">Department of Electronic and Computer Engineering, The Hong Kong University of Science and Technology</h3>
                        <div class="subheading mb-3">Research Associate</div>
                    </div>
                    <div class="resume-date text-md-right">
                        <span class="text-primary">June 2017 - December 2018</span>
                    </div>
                </div>
                <div class="resume-item d-flex flex-column flex-md-row">
                    <div class="resume-content mr-auto">
                        <h3 class="mb-0">Division of Biomedical Engineering, The Hong Kong University of Science and Technology</h3>
                        <div class="subheading mb-3">Doctor of Philosophy</div>
                    </div>
                    <div class="resume-date text-md-right">
                        <span class="text-primary">September 2012 - June 2017</span>
                    </div>
                </div>
                <div class="resume-item d-flex flex-column flex-md-row">
                    <div class="resume-content mr-auto">
                        <h3 class="mb-0">Department of Optical Engineering, Zhejiang University</h3>
                        <div class="subheading mb-3">Master of Engineering</div>
                    </div>
                    <div class="resume-date text-md-right">
                        <span class="text-primary">September 2009 - March 2012</span>
                    </div>
                </div>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="publications">
            <div class="my-auto">
                <h2 class="mb-5">Journal Publications</h2>
                <ol class="mb-0">
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Polasky, D. A., Lu, L., <strong>Yu, F.</strong>, Li, K., Shortreed, M. R., Smith, L. M., Nesvizhskii, A. I. (2024). Quantitative proteome-wide O-glycoproteomics analysis with FragPipe. <em>Analytical and Bioanalytical Chemistry</em>, https://doi.org/10.1007/s00216-024-05382-x
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1007/s00216-024-05382-x" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1007/s00216-024-05382-x" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>

                    <li>
                        <div class="row">
                            <div class="col-9">
                                Kohler, D., Staniak, M., <strong>Yu, F.</strong>, Nesvizhskii, A. I., Vitek, O. (2024). An MSstats workflow for detecting differentially abundant proteins in large-scale data-independent acquisition mass spectrometry experiments with FragPipe processing. <em>Nature Protocols</em>, https://doi.org/10.1038/s41596-024-01000-3
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41596-024-01000-3" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41596-024-01000-3" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Ferreira, H. J., Stevenson, B. J., Pak, H. S., <strong>Yu, F.</strong>, Oliveira, J. A., Huber, F., Taillandier-Coindard, M., Michaux, J., Ricart-Altimiras, E., Kraemer, A. I., Kandalaft, L. E., Speiser, D. E., Nesvizhskii, A. I., Müller, M., Bassani-Sternberg, M. (2024). Immunopeptidomics-based identification of naturally presented non-canonical circRNA-derived peptides. <em>Nature Communications</em>, 15, 2357
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41467-024-46408-3" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-024-46408-3" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Mi, Y., Burnham, K. L., Charles, P. D., Heilig, R., Vendre, I., Whalley, J., Torrance, H. D., Antcliffe, D. B., May, S. M., Neville, M. J., Berridge, G., Hutton, P., Geoghegan, C. G., Radhakrishnan, J., Nesvizhskii, A. I., <strong>Yu, F.</strong>, GAinS Investigators, Davenport, E. E., McKechnie, S., Davies, R., O’Callaghan, D. JP, Pate, P., Karpe, F., Gordon, A. C., Ackland, G. L., Hinds, C. J., Fischer, R., Knight, J. C., (2024). High-throughput mass spectrometry maps the sepsis plasma proteome and differences. <em>Science Translational Medicine</em>, 16(750), eadh0185
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1126/scitranslmed.adh0185" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1126/scitranslmed.adh0185" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Li, G. X., Chen, L., Hsiao, Y., Mannan, R., Zhang, Y., Luo, J., Petralia, F., Cho, H., Hosseini, N., da Veiga Leprevost, F., Calinawan, A., Li, Y., Anand, S., Dagar, A., Geffen, Y., Kumar-Sinha, C., Chugh, S., Le, A., Ponce, S., Guo, S., Zhang, C., Schnaubelt, M., Naser Al Deen, N., Chen, F., Caravan, W., Houston, A., Hopkins, A., Newton, C. J., Wang, X., Polasky, D. A., Haynes, S., <strong>Yu, F.</strong>, Jing, X., Chen, S., Robles, A. I., Mesri, M., Thiagarajan, M., An, E., Getz, G. A., Linehan, W. M., Hostetter, G., Jewell, S. D., Chan, D. W., Wang, P., Omenn, G. S., Mehra, R., Ricketts, C. J., Ding, L., Chinnaiyan, A. M., Cieslik, M. P., Dhanasekaran, S. M., Zhang, H., Nesvizhskii, A. I., & Clinical Proteomic Tumor Analysis Consortium. (2024). Comprehensive proteogenomic characterization of rare kidney tumors. <em>Cell Reports Medicine</em>, 5(5), 101547
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.xcrm.2024.101547" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.xcrm.2024.101547" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>*, Teo, G. C., Kong, A. T., Fröhlich, K., Li, G. X., Demichev, V., &amp; Nesvizhskii, A. I.*, (2023). Analysis of DIA proteomics data using MSFragger-DIA and FragPipe computational platform. <em>Nature Communications</em>, 14, 4154. (* co-corresponding authors)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41467-023-39869-5" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-023-39869-5" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Yang, K. L., <strong>Yu, F.</strong>*, Teo, G. C., Demichev, V., Ralser, M., &amp; Nesvizhskii, A. I.*, (2023). MSBooster: improving peptide identification rates using deep learning-based features. <em>Nature Communications</em>, 14, 4539. (* co-corresponding authors)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41467-023-40129-9" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-023-40129-9" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Geiszler, D. J., Polasky, D. A., <strong>Yu, F.</strong>, &amp; Nesvizhskii, A. I., (2023). Detecting diagnostic features in MS/MS spectra of post-translationally modified peptides. <em>Nature Communications</em>, 14, 4132.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41467-023-39828-0" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-023-39828-0" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Bedran, G., Polasky, D. A., Hsiao, Y., <strong>Yu, F.</strong>, da Veiga Leprevost, F., Alfaro, J. A., Cieslik, M., &amp; Nesvizhskii, A. I., (2023). Unraveling the glycosylated immunopeptidome with HLA-Glyco. <em>Nature Communications</em>, 14, 3461.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41467-023-39270-2" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-023-39270-2" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Polasky, D., Geiszler, D., <strong>Yu, F.</strong>, Li, K., Teo, G. C., &amp; Nesvizhskii, A. I., (2023). MSFragger-Labile: A flexible method to improve labile PTM analysis in proteomics. <em>Molecular &amp; Cellular Proteomics</em>, 22(5), 100538.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.mcpro.2023.100538" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.mcpro.2023.100538" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Park, J., <strong>Yu, F.</strong>, Fulcher, J.M., Williams, S.M., Engbrecht, K., Moore, R.J., Clair, G.C., Petyuk, V., Nesvizhskii, A.I., &amp; Zhu, Y., (2023). Evaluating linear ion trap for MS3-based multiplexed single-cell proteomics. <em>Analytical Chemistry</em>, 95(3), 1888-1898.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.analchem.2c03739" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.analchem.2c03739" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Dai, S., Liu, S., Zhou, C., <strong>Yu, F.</strong>, Zhu, G., Zhang, W., Deng, H., Burlingame, A., Yu, W., Wang, T., Li, N., (2023). Capturing the hierarchically assorted modules of protein-protein interaction in the organized nucleome. <em>Molecular Plant</em>, 16(5), 930-961.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.molp.2023.03.013" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.molp.2023.03.013" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Chang, H.-Y., Haynes, S. E., <strong>Yu, F.</strong>, &amp; Nesvizhskii, A. I., (2022). Implementing the MSFragger search engine as a node in Proteome Discoverer. <em>Journal of Proteome Research</em>, 22(2), 520-525.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.2c00485" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.2c00485" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Ploasky, D., <strong>Yu, F.</strong>, Geislzer, D., Kai, L., &amp; Nesvizhskii, A. I., (2022). Enabling large-scale glycoproteomics data analysis with the MSFragger Glyco software suite. <em>GLYCOBIOLOGY</em>, 32(11), 973-974.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1093/glycob/cwac066" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1093/glycob/cwac066" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Kacen, A., Javitt, A., Kramer, M., Morgenstern, D., Tsaban, T., Shmueli, M., Teo, G. C., da Veiga Leprevost, F., Barnea, E., <strong>Yu, F.</strong>, Admon, A., Eisenbach, L., Samuels, Y., Schueler-Furman, O., Levin, Y., &amp; Nesvizhskii, A. (2022). Post-translational modifications reshape the antigenic landscape of MHC I-immunopeptidome in tumors. <em>Nature Biotechnology</em>, 41(2), 239-251.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41587-022-01464-2" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41587-022-01464-2" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Demichev, V., Szyrwiel, L., <strong>Yu, F.</strong>, Teo, G. C., Rosenberger, G., Niewienda, A., Ludwig, D., Decker, J., Kaspar-Schoenefeld, S., Lilley, K., Mulleder, M., Nesvizhskii, A., &amp; Ralser, M. (2022). dia-PASEF data analysis using FragPipe and DIA-NN for deep proteomics of low sample amounts. <em>Nature Communications</em>, 13, 3944.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41467-022-31492-0" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-022-31492-0" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Desai, H., Yan, T., <strong>Yu, F.</strong>, Sun, A., Villanueva, M., Nesvizhskii, A., &amp; Backus, K. (2022). SP3-enabled rapid and high coverage chemoproteomic identification of cell-state dependent redox-sensitive cysteines. <em>Molecular &amp; Cellular Proteomics</em>, 21(4), 100218.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.mcpro.2022.100218" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.mcpro.2022.100218" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Polasky, D., Geiszler, D., <strong>Yu, F.</strong> &amp; Nesvizhskii, A. (2022). Multiattribute glycan identification and FDR control for glycoproteomics. <em>Molecular &amp; Cellular Proteomics</em>, 21(3), 100205.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.mcpro.2022.100205" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.mcpro.2022.100205" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Moradi, A., Dai, S., Wong, E.O.Y., Zhu, G., <strong>Yu, F.</strong>, Lam, H.M., Wang, Z., Burlingame, A., Lin, C., Afsharifar, A. &amp; Yu, W. (2021). Isotopically dimethyl labeling-based quantitative proteomic analysis of phosphoproteomes of soybean cultivars. <em>Biomolecules</em>, 11(8), p.1218.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.3390/biom11081218" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.3390/biom11081218" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Petyuk, V.A., Yu, L., Olson, H.M., <strong>Yu, F.</strong>, Clair, G., Qian, W.J., Shulman, J.M. &amp; Bennett, D.A. (2021). Proteomic profiling of the substantia nigra to identify determinants of lewy body pathology and dopaminergic neuronal loss. <em>Journal of Proteome Research</em>, 20(5), 2266-2282.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.0c00747" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.0c00747" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Haynes, S., &amp; Nesvizhskii, A. (2021). IonQuant enables accurate and sensitive label-free quantification with FDR-controlled match-between-runs. <em>Molecular &amp; Cellular Proteomics</em>, 20, 100077.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.mcpro.2021.100077" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.mcpro.2021.100077" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Huang, C., Chen, L., Savage, S. R., Eguez, R. V., Dou, Y., Li, Y., ... & Piehowski, P. D. (2021). Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma. <em>Cancer Cell</em>, 39(3), 361-379.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.ccell.2020.12.007" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ccell.2020.12.007" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Geiszler, D., Kong, A., Avtonomov, D., <strong>Yu, F.</strong>, da Veiga Leprevost, F., &amp; Nesvizhskii, A. (2021). PTM-Shepherd: analysis and summarization of post-translational and chemical modifications from open search results. <em>Molecular &amp; Cellular Proteomics</em>, 20, 100018.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1074/mcp.TIR120.002216" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1074/mcp.TIR120.002216" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Dai, J.*, <strong>Yu, F.*</strong>, Chen Zhou, &amp; Yu, W. (2021). Understanding the limit of open search in the identification of peptides with post-translational modifications — A simulation-based study. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em>, 18(6), 2884-2890. (* co-first authors)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1109/TCBB.2020.2991207" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1109/TCBB.2020.2991207" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Teo, G. C., Polasky, D., <strong>Yu, F.</strong>, &amp; Nesvizhskii, A. (2020). Fast deisotoping algorithm and its implementation in the MSFragger search engine. <em>Journal of Proteome Research</em>, 20(1), 498-505.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.0c00544" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.0c00544" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Polasky, D., <strong>Yu, F.</strong>, Teo, G. C., &amp; Nesvizhskii, A. (2020). Fast and comprehensive N- and O-glycoproteomics analysis with MSFragger-Glyco. <em>Nature Methods</em>, 17(11), 1125–1132.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41592-020-0967-9" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41592-020-0967-9" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Teo, G. C., Kong, A., Avtonomov, D., Geiszler, D., &amp; Nesvizhskii, A. (2020). Identification of modified peptides using localization-aware open search. <em>Nature Communications</em>, 11, 4065.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41467-020-17921-y" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-020-17921-y" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.*</strong>, Haynes, S.*, Teo, G. C., Avtonomov, D., Polasky, D., &amp; Nesvizhskii, A. (2020). Fast quantitative analysis of timsTOF PASEF data with MSFragger and IonQuant. <em>Molecular &amp; Cellular Proteomics</em>, 19(9), 1575-1585. (* co-first authors)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1074/mcp.TIR120.002048" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1074/mcp.TIR120.002048" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Liu, S.*, <strong>Yu, F.*</strong>, Hu, Q., Wang, T., Yu, L., Du, S., Yu, W., &amp; Li, N. (2018). Development of in planta chemical cross-linking-based quantitative interactomics in Arabidopsis. <em>Journal of Proteome Research</em>, 17(9), 3195-3213. (* co-first authors)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.8b00320" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.8b00320" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Dai, J.*, Jiang, W.*, <strong>Yu, F.*</strong>, &amp; Yu, W. (2018). Xolik: finding cross-linked peptides with maximum paired scores in linear time. <em>Bioinformatics</em>, 35(2), 251-257. (* co-first authors)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1093/bioinformatics/bty526" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/bty526" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Liu, S.*, <strong>Yu, F.*</strong>, Yang, Z., Wang, T., Xiong, H., Chang, C., Yu, W., &amp; Li, N. (2018). Establishment of dimethyl labeling-based quantitative acetylproteomics in Arabidopsis. <em>Molecular &amp; Cellular Proteomics</em>, 17(5), 1010-1027. (* co-first authors)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1074/mcp.RA117.000530" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1074/mcp.RA117.000530" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (2017). Exhaustively identifying cross-linked peptides with a linear computational complexity. <em>Journal of Proteome Research</em>, 16(10), 3942-3952.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.7b00338" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.7b00338" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (2016). PIPI: PTM-invariant peptide identification using coding method. <em>Journal of Proteome Research</em>, 15(12), 4423-4435.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1021/acs.jproteome.6b00485" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1021/acs.jproteome.6b00485" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (2016). ECL: an exhaustive search tool for the identification of cross-linked peptides using whole database. <em>BMC Boinformatics</em>, 17(1), 217.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1186/s12859-016-1073-y" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1186/s12859-016-1073-y" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                Zhu, X.*, <strong>Yu, F.*</strong>, Yang, Z., Liu, S., Dai, C., Lu, X., Liu, C., Yu, W., &amp; Li, N. (2016). In planta chemical cross‐linking and mass spectrometry analysis of protein structure and interaction in Arabidopsis. <em>Proteomics</em>,
                                16(13), 1915-1927. (* co-first authors)
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1002/pmic.201500310" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1002/pmic.201500310" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-9">
                                <strong>Yu, F.</strong>, Liu, H., Hu, Z., &amp; Shi, P. (2012). Graphics processing unit (GPU)-accelerated particle filter framework for positron emission tomography image reconstruction. <em>JOSA A</em>, 29(4), 637-643.
                            </div>
                            <div class="col-1 mb-2">
                                <div data-badge-popover="left" data-badge-type="donut" data-doi="10.1364/JOSAA.29.000637" data-hide-no-mentions="true" class="altmetric-embed"></div>
                            </div>
                            <div class="col-1 mb-2">
                                <span class="__dimensions_badge_embed__" data-doi="10.1364/JOSAA.29.000637" data-hide-zero-citations="true" data-style="small_circle"></span>
                            </div>
                        </div>
                    </li>
                </ol>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="conference">
            <div class="my-auto">
                <h2 class="mb-5">Invited Oral Presentations</h2>
                <ol class="mb-0">
                    <li>
                        <strong>Yu, F.</strong> (August 30-31, 2023). FragPipe enables the one-stop analysis for DDA and DIA bottom-up proteomics. <em>The 7th China Workshop on Computational Proteomics (CNCP 2023)</em>. Beijing, China.
                    </li>
                    <li>
                        <strong>Yu, F.</strong> (August 24, 2022). <a href=https://www.cancergenomicscloud.org/new-events/2022/8/24/webinar-august-24-2pm-et-l7n6p-tzr3d-abykd target=_blank>FragPipe enables one-stop proteomics data analysis</a>. <em>Cancer Genomics Cloud Webinar</em>.
                    </li>
                </ol>
                <br>
                <br>
                <br>
                <h2 class="mb-5">Workshops</h2>
                <ol class="mb-0">
                    <li>
                        Kulej, K., <strong>Yu, F.</strong> (June 5, 2024). Wed 01 Integrative proteogenomic approach for personalized protein mapping: prospects, challenges, and scalability bottlenecks. <em>ASMS 2024, evening workshops</em>. Anaheim, California.
                    </li>
                </ol>
                <br>
                <br>
                <br>
                <h2 class="mb-5">Conference Presentations</h2>
                <ol class="mb-0">
                    <li>
                        <strong>Yu, F.</strong>, Deng, Y., Polasky, D. A., Yang, K. L., Li, K., Teo, G. C., & Nesvizhskii, A. I. (June 2-6, 2024). FragPipe Advancements for Optimized DDA, DDA+, and DIA Mass Spectrometry Data Analysis. <em>72nd ASMS Conference on Mass Spectrometry and Allied Topics (ASMS 2024)</em>. Anaheim, California. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong> (June 2, 2024). Integration of Skyline into FragPipe for Streamlined Visualization. <em>2024 User Group Meeting at ASMS</em>. Anaheim, California. (oral presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Yang, K. L., Teo, G. C., Li, K., & Nesvizhskii, A. I. (March 9-13, 2024). The development of MSFragger-DDA+ and MSFragger-DIA for multiplexed DDA and DIA data analysis. <em>20th Annual US HUPO Conference (US HUPO 2024)</em>. Portland, Oregon. (oral presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Polasky, D. A., Yang, K. L., Deng, Y., Teo, G. C., & Nesvizhskii, A. I. (October 13-16, 2023). FragPipe Enhancements: Integrating MSFragger-DIA and MSFragger-WWA for Advanced Single-Cell and Bulk-Cell Proteomics Analysis. <em>38th Asilomar Conference on Mass Spectrometry</em>. Pacific Grove, California (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Truong, T., Kelly, R., &amp; Nesvizhskii, A. I. (September 17-21, 2023). MSFragger-WWA coupled with FragPipe enables fast and easy wide-window acquisition data analysis. <em>22th Human Proteome Organization World Congress (HUPO 2023)</em>. Busan, South Korea. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Polasky, D. A., Yang, K. L., Kong, A. T., & Nesvizhskii, A. I. (August 28-31, 2023). MSFragger-DIA and MSFragger-WWA facilitate fast and easy single-cell proteomics data analysis. <em>3rd CASMS Virtual Conference</em>. Virtual. (poster presentation, best poster award).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Polasky, D. A., Kong, A. T., Teo, G. C., Yang, K. L., &amp; Nesvizhskii, A. I. (June 4-8, 2023). Updates in the MSFragger search engine: facilitating bulk-cell and single-cell proteomics data analysis. <em>71th ASMS Conference on Mass Spectrometry and Allied Topics (ASMS 2023)</em>. Houston, Texas. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Li, K., Yang, K., Polasky, D., &amp; Nesvizhskii, A. (March 4-8, 2023). Comprehensive single-cell proteomics data analysis using FragPipe. <em>19th Annual US HUPO Conference (US HUPO 2023)</em>. Chicago Illinois. (oral presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Teo, G.C., da Veiga Leprevost, F., Polasky, D., Geiszler, D., Yang, K., Li, K., &amp; Nesvizhskii, A. (October 17-21, 2022). FragPipe enables one-stop data analysis for bottom-up proteomics. <em>3rd CASMS Virtual Conference</em>. Virtual. (poster presentation, best poster award).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Teo, G.C., Avtonomov. D., Haynes, S., da Veiga Leprevost, F., Geiszler, D., Polasky, D., Yang, K., &amp; Nesvizhskii, A. (July 17-23, 2022). FragPipe in headless mode enables seamlessly analyzing LC-MS data in Galaxy platform. <em>2022 Galaxy Community Conference (GCC 2022)</em>. Minneapolis, Minnesota. (oral presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Kong, A., Patil, S., Avtonomov, D., &amp; Nesvizhskii, A. (June 5-9, 2022). Pepcentric enables fast peptide searching against public data repositories. <em>70th ASMS Conference on Mass Spectrometry and Allied Topics (ASMS 2022)</em>. Minneapolis, Minnesota. (poster presentation, <a href=http://pepcentric.arsci.com:8080 target=_blank>http://pepcentric.arsci.com:8080</a>).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Yang, K., Demichev, V., Haynes, S., Ralser, M., &amp; Nesvizhskii, A. (Feburary 26 - March 2, 2022). One-stop DIA data analysis using MSFragger-DIA coupled with FragPipe. <em>18th Annual US HUPO Conference (US HUPO 2022)</em>. Charleston, Sourth Carolina. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Teo, G. C., Haynes, S., Li, G. X., &amp; Nesvizhskii, A. (October 31 - November 4, 2021). Direct peptide identification from DIA data with MSFragger DIA. <em>69th ASMS Conference on Mass Spectrometry and Allied Topics (ASMS 2021)</em>. Philadelphia, Pennsylvania. (oral presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Haynes, S., &amp; Nesvizhskii, A. (March 4-12, 2021). False discovery rate-controlled match-between-runs enables accurate and sensitive label free quantification. <em>17th Annual US HUPO Conference (US HUPO 2021)</em>. Virtual. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Teo, G. C., Kong, A., Haynes, S., Avtonomov, D., Geiszler, D., &amp; Nesvizhskii, A. (July 13-16, 2020). Improved identification of modified peptides using localization-aware open search. <em>28th Conference on Intelligent Systems for Molecular Biology (ISMB 2020)</em>. Virtual. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Haynes, S., Avtonomov, D., Kong, A., da Veiga Leprevost, F., &amp; Nesvizhskii, A. (June 1-12,2020). Breaking the logjam: fast peptide identification and quantification in timsTOF PASEF data. <em>68th ASMS Conference on Mass Spectrometry and Allied Topics (ASMS 2020)</em>. Virtual. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Haynes, S., Teo, G. C., Kong, A., Avtonomov, D., da Veiga Leprevost, F., Chang, H.-Y., Geiszler, D., Polasky, D., &amp; Nesvizhskii, A. (March 6-10, 2020). Fast and quantitative analysis of timsTOF PASEF data with MSFragger and IonQuant. <em>16th Annual US HUPO Conference (US HUPO 2020)</em>. Virtual. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Teo, G. C., Kong, A., da Veiga Leprevost, F., Avtonomov, D., Chang, H., Geiszler, D., Haynes, S., Polasky, D., &amp; Nesvizhskii, A. (September 15-19, 2019). MSFragger: fast and sensitive peptide identification in diverse proteomic datasets. <em>18th Human Proteome Organization World Congress (HUPO 2019)</em>. Adelaide, Australia. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Teo, G. C., Kong, A., da Veiga Leprevost, F., Chang, H., &amp; Nesvizhskii, A. (June 2-6, 2019). Comparison of open search tools. <em>67th ASMS Conference on Mass Spectrometry and Allied Topics (ASMS 2019)</em>. Atlanta, Georgia. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (August 22-23, 2018). PTM-invariant peptide identification. <em>5th China Workshop on Computational Proteomics (CNCP 2018)</em>. Beijing China, (oral presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (August 13-18, 2017). PIPI: PTM-invariant peptide identification using coding method. <em>Post-translational Modification Networks, Gordon Research Conference</em>. Hong Kong, China. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (August, 2017). PIPI: PTM-invariant peptide identification using coding method. <em>11th International Conference on Computational Systems Biology</em>. Shenzhen, China. (oral presentation)
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Li, N., &amp; Yu, W. (May 3-7, 2017). Exhaustively identifying cross-linked peptides with a linear complexity. <em>21st Annual International Conference on Research in Computational Molecular Biology (RECOMB 2017)</em>. Hong Kong, China. (poster presentation).
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Liu, H., &amp; Shi, P. (January 5-7, 2012). PET image reconstruction based on particle filter framework. <em>In Biomedical and Health Informatics (BHI), 2012 IEEE-EMBS International Conference on</em> (pp. 851-853). IEEE.
                    </li>
                    <li>
                        <strong>Yu, F.</strong>, Liu, H., &amp; Shi, P. (September 11-14, 2011). PET image reconstruction: GPU-accelerated particle filter framework. <em>In Image Processing (ICIP), 2011 18th IEEE International Conference on</em> (pp. 417-420). IEEE.
                    </li>
                </ol>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="awards">
            <div class="my-auto">
                <h2 class="mb-5">Awards</h2>
                <ul class="fa-ul mb-0">
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>CASMS (Chinese American Society for Mass Spectrometry) Best Poster Award (2023)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>CASMS (Chinese American Society for Mass Spectrometry) Best Poster Award (2022)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>US HUPO (United States Human Proteome Organization) Travel Stipend Award (2020)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>UGC Research Travel Grant Award (2018-19)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>UGC Research Travel Grant Award (2018-19)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>UGC Research Travel Grant Award (2017-18)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>UGC Research Travel Grant Award (2016-17)
                    </li>
                    <li>
                        <i class="fa-li fa fa-trophy text-warning"></i>Graduate Student Research Award Program of HKMHDIA (Hong Kong Medical &amp; Healthcare Device Industries Association Limited) (2015-16)
                    </li>
                </ul>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="activities">
            <div class="my-auto">
                <h2 class="mb-5">Professional Activities</h2>
                <h3 class="mb-5">Journal reviewer</h3>
                <style type="text/css">
                .tg {
                    border-collapse: collapse;
                    border-spacing: 0;
                }

                .tg td {
                    border-color: black;
                    border-style: solid;
                    border-width: 1px;
                    font-family: Arial, sans-serif;
                    font-size: 14px;
                    overflow: hidden;
                    padding: 10px 5px;
                    word-break: normal;
                }

                .tg th {
                    border-color: black;
                    border-style: solid;
                    border-width: 1px;
                    font-family: Arial, sans-serif;
                    font-size: 14px;
                    font-weight: normal;
                    overflow: hidden;
                    padding: 10px 5px;
                    word-break: normal;
                }

                .tg .tg-7zrl {
                    text-align: left;
                    vertical-align: bottom
                }
                </style>
                <table class="tg">
                    <thead>
                        <tr>
                            <th class="tg-7zrl">Journal</th>
                            <th class="tg-7zrl">Years</th>
                            <th class="tg-7zrl">Reviewed</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td class="tg-7zrl">Nature Communications</td>
                            <td class="tg-7zrl">2021, 2022</td>
                            <td class="tg-7zrl">3</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Nature Computational Science</td>
                            <td class="tg-7zrl">2020</td>
                            <td class="tg-7zrl">3</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Scientific Data</td>
                            <td class="tg-7zrl">2023</td>
                            <td class="tg-7zrl">2</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Briefings in Bioinformatics</td>
                            <td class="tg-7zrl">2020, 2021, 2022</td>
                            <td class="tg-7zrl">12</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Bioinformatics</td>
                            <td class="tg-7zrl">2019, 2021, 2023, 2024</td>
                            <td class="tg-7zrl">14</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Analytical Chemistry</td>
                            <td class="tg-7zrl">2020, 2021, 2023</td>
                            <td class="tg-7zrl">9</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Journal of Proteome Research</td>
                            <td class="tg-7zrl">2019, 2020, 2021, 2022, 2023, 2024</td>
                            <td class="tg-7zrl">16</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Proteomics</td>
                            <td class="tg-7zrl">2019, 2020</td>
                            <td class="tg-7zrl">5</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Journal of Proteomics</td>
                            <td class="tg-7zrl">2021</td>
                            <td class="tg-7zrl">2</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">ACS Omega</td>
                            <td class="tg-7zrl">2021</td>
                            <td class="tg-7zrl">1</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">BMC Bioinformatics</td>
                            <td class="tg-7zrl">2016, 2019, 2020</td>
                            <td class="tg-7zrl">8</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">BMC Genomics</td>
                            <td class="tg-7zrl">2019</td>
                            <td class="tg-7zrl">1</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Rapid Communications in Mass Spectrometry</td>
                            <td class="tg-7zrl">2016</td>
                            <td class="tg-7zrl">1</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Bioinformatics Advances</td>
                            <td class="tg-7zrl">2021, 2023</td>
                            <td class="tg-7zrl">5</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Communications Biology</td>
                            <td class="tg-7zrl">2023</td>
                            <td class="tg-7zrl">1</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Genomics, Proteomics & Bioinformatics</td>
                            <td class="tg-7zrl">2024</td>
                            <td class="tg-7zrl">1</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Journal of American Society for Mass Spectrometry</td>
                            <td class="tg-7zrl">2024</td>
                            <td class="tg-7zrl">2</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">International Journal of Mass Spectrometry</td>
                            <td class="tg-7zrl">2022</td>
                            <td class="tg-7zrl">2</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">Small Methods</td>
                            <td class="tg-7zrl">2024</td>
                            <td class="tg-7zrl">1</td>
                        </tr>
                        <tr>
                            <td class="tg-7zrl">total:</td>
                            <td class="tg-7zrl"></td>
                            <td class="tg-7zrl">89</td>
                        </tr>
                    </tbody>
                </table>
                <br><br><br><br>
                <h3 class="mb-5">Conference Committee</h3>
                <style type="text/css">
                .tg {
                    border-collapse: collapse;
                    border-spacing: 0;
                }

                .tg td {
                    border-color: black;
                    border-style: solid;
                    border-width: 1px;
                    font-family: Arial, sans-serif;
                    font-size: 14px;
                    overflow: hidden;
                    padding: 10px 5px;
                    word-break: normal;
                }

                .tg th {
                    border-color: black;
                    border-style: solid;
                    border-width: 1px;
                    font-family: Arial, sans-serif;
                    font-size: 14px;
                    font-weight: normal;
                    overflow: hidden;
                    padding: 10px 5px;
                    word-break: normal;
                }

                .tg .tg-7zrl {
                    text-align: left;
                    vertical-align: bottom
                }
                </style>
                <table class="tg">
                    <thead>
                        <tr>
                            <th class="tg-7zrl">Conferences</th>
                            <th class="tg-7zrl">Years</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td class="tg-7zrl">ISMB CompMS program committee</td>
                            <td class="tg-7zrl">2023, 2024</td>
                        </tr>
                    </tbody>
                </table>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="teaching">
            <div class="my-auto">
                <h2 class="mb-5">Teaching Activities</h2>
                <ul class="fa-ul mb-0">
                    <li>
                        <i class="fa-li fa fa-solid fa-chalkboard-user text-warning"></i>US HUPO 2023 short course: "Design and Analysis of Quantitative Proteomics Experiments with TMT Labeling: case studies using FragPipe and MSstatsTMT", May 4-5, 2023
                    </li>
                    <li>
                        <i class="fa-li fa fa-solid fa-chalkboard-user text-warning"></i>University of Michigan, BIOINF 551 FA 2022 (teaching assistant)
                    </li>
                    <li>
                        <i class="fa-li fa fa-solid fa-chalkboard-user text-warning"></i>EMBO practical course: <a href="https://www.embl.org/about/info/course-and-conference-office/events/qpr21-01/#vf-tabs__section-311a4dae-4712-45c4-bb7c-83f569003b70">"Quantitative Proteomics: Strategies and Tools to Probe Biology"</a>, May 3-7, 2021
                    </li>
                </ul>
            </div>
        </section>
    </div>
    <script src="js/jquery-3.3.1.min.js"></script>
    <script src="js/jquery.easing.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <script src="js/popper.min.js"></script>
    <script src="js/resume.min.js"></script>
</body>

</html>
